recipes/datashader/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "datashader" %}
+{% set version = "0.6.2" %}
+{% set sha256 = "fc76c791fc050b9564e6ecd57095ce42d1c10b11bbee751c796431b70daa85aa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/bokeh/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Any thoughts on adding `noarch: python`?

Reply:
Happy to have noarch: Python; I strongly believe in noarch!

Reply:
Cool, well let's add that and then I think this is good to go. :)